CHANGELOG.md

```diff
@@ -34,6 +34,7 @@ Note that this project **does not** adhere to [Semantic Versioning](https://semv
 - We added a different background color to the search bar to indicate when the search syntax is wrong. [#11658](https://github.com/JabRef/jabref/pull/11658)
 - We added a setting which always adds the literal "Cited on pages" text before each JStyle citation. [#11691](https://github.com/JabRef/jabref/pull/11732)
 - We added a new plain citation parser that uses LLMs. [#11825](https://github.com/JabRef/jabref/issues/11825)
+- We added an importer for ssrn URLs. [#12021](https://github.com/JabRef/jabref/pull/12021)
```

Comment:
Accoarding to https://www.ssrn.com/index.cfm/en/, SSRN is written with capital letters:

```suggestion
- We added an importer for SSRN URLs. [#12021](https://github.com/JabRef/jabref/pull/12021)
```